sig-advocacy-and-outreach/gsoc/README.md

```diff
@@ -1,5 +1,10 @@
-KubeSphere in [Google Summber of Code (AKA GSoC)](https://summerofcode.withgoogle.com/). You can find our propsosal in this page. Or you also can submit your proposal to KubeSphere community.
+KubeSphere in [Google Summber of Code (aka GSoC)](https://summerofcode.withgoogle.com/). You can find our proposals in this page. Or you also can submit your proposal to KubeSphere community.
 
 ## Proposals
 
-TODO
+| Project | Area | Skills to study/improve |
+| --- | --- | --- |
+| [KubeSphere Jenkins Client Refactor](kubeSphere-jenkins-client-refactor.md) <br/>Split the Jenkins client from KubeSphere. Or use an existing Jenkins client. </br>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, REST API/OpenAPI, Jenkins, Kubernetes |
+| [KubeSphere Tekton Integration](kubeSphere-tekton-integration.md) <br/>Integrate Tekton as an alternative CI/CD engine of KubeSphere DevOps. <br/>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, Tekton, Jenkins, Kubernetes |
```

Comment:
@Felixnoo Please help to refine these two proposals: KubeSphere Jenkins Client Refactor and KubeSphere Tekton Integration, you would better communicate with @LinuxSuRen to make it much clear.